ui/src/widgets/ui-gauge/types/UIGaugeDial.vue

```diff
@@ -77,12 +77,27 @@ export default {
         },
         iconOnly () {
             return this.props.icon && !this.props.units
+        },
+        gaugeStyle () {
+            return this.props.gstyle
+        },
+        updateOn () {
+            return `${this.segments} ${this.gaugeStyle} ${this.min} ${this.max}`
```

Comment:
check for changes in segments, gauge styles, minimum values and maximum values and watching for changes in line 100